node/middlewares/stockFilter.ts

```diff
@@ -0,0 +1,8 @@
+export async function stockFilterEvents(ctx: Context, next: () => Promise<void>) {
+  // Temporary if, in the near future broadcaster will only notify us modifications in SKUs or product.
```

Comment:
You can remove this comment, it is outdated

Reply:
Ok, ty